package.json

```diff
@@ -129,7 +129,7 @@
     "express": "^4.17.1",
     "faker": "^5.2.0",
     "file-loader": "^6.2.0",
-    "husky": "^4.3.8",
+    "husky": "^5.1.3",
```

Comment:
Why did this get updated?

Reply:
The lock file should also have no changes

Reply:
I made the update locally, but I will revert to the 4.3.8 version.